Lib/fontTools/feaLib/lexer.py

```diff
@@ -193,14 +195,14 @@ def __init__(self, featurefile):
     def __iter__(self):
         return self
 
-    def next(self):  # Python 2
-        return self.__next__()
+    def next(self, comments = False):  # Python 2
```

Comment:
one of the reason we didn't add an automatic PEP8 checker (e.g. flake8) to the CI is because unfortunately half of the code base will not pass the test... :)
But we try to ensure that new code which is added follows it as far as possible, without being overly strict. The spaces around the equal sign look a bit odd to me as well.